pkg/catalog/routes.go

```diff
@@ -572,6 +582,18 @@ func (mc *MeshCatalog) buildInboundPolicies(t *access.TrafficTarget) []*trafficp
 	return inboundPolicies
 }
 
+func buildInboundPolicyRules(servicePolicy *trafficpolicy.InboundTrafficPolicy, routeMatches []trafficpolicy.HTTPRouteMatch, sourceSvcAccounts []service.K8sServiceAccount, services []service.MeshService) *trafficpolicy.InboundTrafficPolicy {
```

Comment:
What do you think about moving this func to pkg/trafficpolicy and hanging this as a method on *trafficpolicy.InboundTrafficPolicy?

Reply:
I did think of that but `getDefaultWeightedClusterForService` is component of catalog and will not be available to use in pkg/trafficpolicy. Further catalog cannot be imported in trafficpolicy as it will cause circular dependency